.travis.yml

```diff
@@ -14,10 +14,13 @@ cache:
 stages:
   - Build
   - Test
-  - "Release / Deploy"
+  - Release
+  - AfterRelease
+  - Deploy
 
 env:
   global:
+    - FORCE_DEPLOY_TAG=null # Change to taskana version like: v1.0.5-SNAPSHOT to force the deployment.
```

Comment:
We actually have an issue here: This only works if the given version is saved in the travis cache or is the current version.
We can either leave it like this and have this as a convention or read the current version out of the pom file (either via bash or mvn itself)

I prefer the ladder since it creates less mistakes.

Code to extract version: 

<code>VERSION=\`mvn -q -Dexec.executable="echo" -Dexec.args='${project.version}' --non-recursive org.codehaus.mojo:exec-maven-plugin:1.6.0:exec -f ../lib\`</code>

(only working from ci folder)

Reply:
Ok I will use it, this way we won't need to specify the version for forcing deployment